stable/busybox/values.yaml

```diff
@@ -0,0 +1,21 @@
+# Default values for busybox.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: busybox
+  tag: latest
+  pullPolicy: IfNotPresent
+service:
+  name: busybox
+  type: ClusterIP
+  externalPort: 80
+  internalPort: 80
+resources:
+  limits:
+    cpu: 100m
+    memory: 128Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
```

Comment:
These resources don't appear to be used. [Here's](https://github.com/kubernetes/charts/blob/master/stable/datadog/templates/daemonset.yaml#L20) an example of a chart using them.